src/views/Receiving.vue

```diff
@@ -5,85 +5,80 @@
         <ion-title>{{ $t("Receiving") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-    
     <ion-content :fullscreen="true">
       <div>
-        <ion-searchbar :placeholder="$t('Scan ASN to start receiving')"></ion-searchbar>
-        <ion-list>
-          <ion-item @click="() => router.push('/shipment')">
-            <ion-label>
-              <h2>SHPMNT_001</h2>
-              <p>3 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_002</h2>
-              <p>2 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_003</h2>
-              <p>6 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_004</h2>
-              <p>4 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_005</h2>
-              <p>1 {{ $t("item") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-        </ion-list>
+        <ion-searchbar :placeholder="$t('Scan ASN to start receiving')"/>
+
+        <shipment-list-item v-for="shipment in shipments" :key="shipment.productId" :shipment="shipment"/>
```

Comment:
Use CamelCase naming to render the component.
```suggestion
        <ShipmentListItem v-for="shipment in shipments" :key="shipment.productId" :shipment="shipment"/>
```

Reply:
Done